Documentation/ceph-monitoring.md

```diff
@@ -38,7 +38,7 @@ With the Prometheus operator running, we can create a service monitor that will
 From the root of your locally cloned Rook repo, go the monitoring directory:
 
 ```console
-$ git clone --single-branch --branch {{ branchName }} https://github.com/rook/rook.git
+$ git clone --single-branch --branch v1.6.0-beta.0 https://github.com/rook/rook.git
```

Comment:
Do these change back to `{{ branchName }}` after we get to `v1.6.0`?

Reply:
With every patch release we search in the docs and example manifests for the previous release version and replace with the new release version. So this will change to v1.6.0 when we GA, but the `{{ branchName }}` is basically only used in master.